packages/server/src/generate.ts

```diff
@@ -3,7 +3,7 @@ import {
   FileError,
   FileExport,
   searchCodebase,
-} from "@prodo-ai/snoopy-search";
+} from "@prodo-ai/snoopy-search/src"; // tslint:disable-line:no-submodule-imports
```

Comment:
I'm fairly sure this won't work when you're using a built version of the package.